lib/main_common.pm

```diff
@@ -2920,6 +2920,7 @@ sub load_transactional_role_tests {
     # ^ runs only outside of stagings, clear repos otherwise
     loadtest "update/zypper_clear_repos" if is_staging;
     loadtest 'transactional/filesystem_ro';
+    loadtest 'transactional/trup_install_tar' if is_leap("15.3+");
```

Comment:
could you please create new schedule file for this scenario? we don't use main logic anymore.